signup-service/src/main/java/io/mosip/signup/controllers/WebSocketController.java

```diff
@@ -69,6 +72,12 @@ public void onConnected(SessionConnectedEvent connectedEvent) {
     public void onDisconnected(SessionDisconnectEvent disconnectEvent) {
         String username = Objects.requireNonNull(disconnectEvent.getUser()).getName();
         log.info("WebSocket Disconnected >>>>>> {}", username);
+        if(disconnectEvent.getCloseStatus()!=null && !disconnectEvent.getCloseStatus().equals(CloseStatus.NORMAL)){
+            IdentityVerificationTransaction transaction =
+                    cacheUtilService.getVerifiedSlotTransaction(username.split(VALUE_SEPARATOR)[1]);
```

Comment:
As we are splitting the "value" multiple times, can we split it once and assign it to a variable, and use the variable instead?

Reply:
And I don't see any Junit test case change or added.